loader/loader.go

```diff
@@ -174,7 +174,11 @@ func (w *Worker) run(ctx context.Context, fileJobQueue chan *fileJob, runFatalCh
 				if err != nil {
 					// expect pause rather than exit
 					err = terror.WithScope(terror.Annotatef(err, "file %s", job.file), terror.ScopeDownstream)
-					runFatalChan <- unit.NewProcessError(pb.ErrorType_ExecSQL, err)
+					if utils.IsContextCanceledError(err) {
+						runFatalChan <- nil
```

Comment:
Is this needed?

Reply:
If we don't deal with `runFatalChan`, the user will still get cancel error from dmctl. Shall we revise `unit.NewProcessError` to filter that instead of this?

Reply:
if err is context cancel error, don't need to send nil to `runFatalChan`